shared/src/payload/mod.rs

```diff
@@ -66,3 +67,35 @@ pub struct OkexBtcUsdSwapPositionPayload {
     pub signed_usd_exposure: SyntheticCentExposure,
 }
 crate::payload! { OkexBtcUsdSwapPositionPayload, "position.okex.btc-usd-swap" }
+
+/// Payload of snapshot of an order book
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct OrderBookPayload {
+    pub asks: BTreeMap<PriceRaw, QuantityRaw>,
+    pub bids: BTreeMap<PriceRaw, QuantityRaw>,
+    pub timestamp: TimeStamp,
+    pub exchange: ExchangeIdRaw,
+}
+
+/// Message to transmit order book payload
```

Comment:
This is a wrapper for okex specific order book. This documentation is redundant with line 71